web-portal/components/admin-venue-edit/AdminEventsListRow.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <tr>
+    <td>
+      <img
+        v-if="calendar_event.reviewed_by_org"
+        :src="ownerLogo"
+        width="30"
+      />
+
+    </td>
+
+    <td>{{ calendar_event.title }}</td>
+
+    <td>
+      <div v-for="(date_time, index) in calendar_event.date_times.slice(0, 2)" :key="index">
+        {{ dateFormat(date_time) }}
+      </div>
+
+      <small v-if="calendar_event.date_times.length > 2">
+        (and {{ calendar_event.date_times.length - 2 }} more)
+      </small>
+
+      <template
+        v-if="calendar_event.category && calendar_event.category==='online-resource'"
+      >
+        Online Resource
+      </template>
+    </td>
+
+    <td>
+      <ii-form-button
+        style-type="light"
+        @click="onEditClicked(calendar_event.id)"
+        :test-id="'edit-issue-button-' + calendar_event.id"
+      >
+        Edit
+      </ii-form-button>
+    </td>
+
+    <td>
+      <input
+        type="checkbox"
+        :checked="isProblem"
+        @change="hasProblemsValueUpdated(calendar_event, $event)"
+      >
+    </td>
+  </tr>
+</template>
+
+<script>
+  import FormButton from '@/components/FormButton.vue'
+  import momenttz from 'moment-timezone'
+  import PartnerService from '@/services/PartnerService'
+
+  export default {
+    name: 'AdminEventsListRow',
+    components: {
+      'ii-form-button': FormButton
+    },
+    props: ['calendar_event', 'isProblemUpdated'],
```

Comment:
We use `calendar_event` for prop name elsewhere, but it breaks Vue's style guide and I'd like to move away from it at some point. Thoughts on either `calendarEvent` (which would be `calendar-event="source"` in the template) or just `event`?

Reply:
:heavy_check_mark: